prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -41,33 +43,79 @@ private const val sheetName = "LOINC Mapping"
  * LivdTableDownload is the command line interface for the livd-table-download command. It parses the command line
  * for option given as below.
  */
-class LivdTableDownload : CliktCommand(
+class LivdTableDownload() : CliktCommand(
     name = "livd-table-download",
     help = """
-    livd-table downloads the latest LOINC test data, so it can be ingested automatically. 
+    livd-table-download downloads the latest LOINC test data from CDC website, so it can be used to update the 
+    new version of the lookup table in the database automatically. 
     
-    It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from $cdcLOINCTestCodeMappingPageUrl.
-	If the file is found, it downloads the file into the directory specified by the --output-dir <path> option.
-	If the option is not specified, it will download the file to ./build directory.
+    It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from $cdcLOINCTestCodeMappingPageUrl.  If the file is found, 
+    it downloads the file into the ./build directory.  If not found, it will prompt error accordingly.  Next, it build 
+    the output Lookup Table (<tableName.csv> file) with the table name specified by --output-file <path/filename> 
+    option. If the --output-file is not specified, it will use the ./metadata/tables/$livdSARSCov2File-latest.csv as
+    the default output file.  Finally, it uses the output filename as the new lookup table name to update the 
+    lookup tables in the database as the new version of the table.  It updates new version of the lookup table in 
+    the given --env [local, test, staging, or prod] with the default to "local" environment.
+    
+    Note, this command will always create new version of the lookup table.
         
     Example: The following command will download the latest LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx from the above URL.  
-      It will store the LIVD-SARS-CoV-2-xyz.csv file under the ./junk directory.
+      It will store the LIVD-SARS-CoV-2-xyz.csv file under the ./junk directory and update the LIVD-SARS-CoV-2-xyz
+      lookup table in the database.
       
          ./prime livd-table-download --output-file ./junk/LIVD-SARS-CoV-2-xyz.csv
+         
+      The command below will download the latest LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx file from the above UTL.
+      It will crate the LIVD-SARS-CoV-2-latest.csv file under the ./metadata/tables directory.
+      
+        ./prime livd-table-download
     """
 ) {
     private val defaultOutputDir = "./build"
+    private val defaultOutPutFile = "./metadata/tables/$livdSARSCov2File-latest.csv"
     private val outputFile by option(
```

Comment:
I also don't think we need output file anymore.  You can potentially change this to an optional option to write the output to a file instead of OR at the same time.  Not sure of a use case here.

Reply:
Removed outputFile option.